### PR TITLE
[Bouffalolab] Update application lighting operation in lighting-app

### PR DESCRIPTION
**Pull request by wy-hh**
Update application lighting operation in lighting-app.

### CODE REVIEW

File: examples/lighting-app/bouffalolab/common/AppTask.cpp [hunk 1]
```diff
@@ -146,7 +138,8 @@ void AppTask::PostEvent(app_event_t event)
 void AppTask::AppTaskMain(void * pvParameter)
 {
     app_event_t appEvent;
-    bool isStateReady = false;
+    bool onoff = false;
+    ;
```
Comment (bzbarsky-apple) [inline examples/lighting-app/bouffalolab/common/AppTask.cpp:142]: Why the extra `';'`?

File: examples/lighting-app/bouffalolab/common/AppTask.cpp [hunk 2]
```diff
@@ -159,8 +152,17 @@ void AppTask::AppTaskMain(void * pvParameter)
     ef_get_env_blob(APP_REBOOT_RESET_COUNT_KEY, &resetCnt, sizeof(resetCnt), &saved_value_len);
     resetCnt++;
     ef_set_env_blob(APP_REBOOT_RESET_COUNT_KEY, &resetCnt, sizeof(resetCnt));
+
+    GetAppTask().mButtonPressedTime = chip::System::SystemClock().GetMonotonicMilliseconds64().count() + 1;
```
Comment (bzbarsky-apple) [inline examples/lighting-app/bouffalolab/common/AppTask.cpp:156]: Please document why the +1 is there.  Yes, I know you just moved this code.

**Review by bzbarsky-apple — APPROVED** [review]

File: examples/lighting-app/bouffalolab/common/AppTask.cpp (hunk 2 above)
Comment (wy-hh) [reply examples/lighting-app/bouffalolab/common/AppTask.cpp:157]: Add comments for this.

File: examples/lighting-app/bouffalolab/common/AppTask.cpp (hunk 1 above)
Comment (wy-hh) [reply examples/lighting-app/bouffalolab/common/AppTask.cpp]: Removed

**Review by jmartinez-silabs — APPROVED** [review]

**Pull request merged**